map/bookmark_manager.cpp

```diff
@@ -404,6 +404,59 @@ void BookmarkManager::ResetRecentlyDeletedBookmark()
   m_recentlyDeletedBookmark.reset();
 }
 
+size_t BookmarkManager::GetRecentlyDeletedCategoriesCount() const
+{
+  Platform::FilesList files;
+  Platform::GetFilesByExt(GetTrashDirectory(), kKmlExtension, files);
+  return files.size();
+}
+
+BookmarkManager::KMLDataCollectionPtr BookmarkManager::GetRecentlyDeletedCategories()
+{
+  auto collection = LoadBookmarks(GetTrashDirectory(), kKmlExtension, KmlFileType::Text,
+                                  [](kml::FileData const &)
+                                  {
+    return true;
+  });
```

Comment:
Will this work?
```suggestion
  auto collection = LoadBookmarks(GetTrashDirectory(), kKmlExtension, KmlFileType::Text, [](kml::FileData const &), {});
```

Reply:
No. The method was designed to work with the `  using BookmarksChecker = std::function<bool(kml::FileData const &)>;`
that should returns bool:
<img width="1183" alt="image" src="https://github.com/user-attachments/assets/17737cb6-ffcc-4f27-b2e5-e2ac5a5c0e42">

Reply:
Sorry for the typo, of course I meant an empty std::function/lambda, which compiles:
`auto collection = LoadBookmarks(GetTrashDirectory(), kKmlExtension, KmlFileType::Text, {});`